docs/product/integrations/integration-platform/webhooks/issues.mdx

```diff
@@ -39,6 +39,23 @@ Sentry integrations that have subscribed to issue webhooks can receive informati
 
 ## Payload
 
+Payload includes additional information about the issue. Some important fields to pay attention to are:
+
+### status
+Some issue webhooks are triggered on status change of an issue. An issue status can be `resolved`, `unresolved`, or `ignored` (archived).
+
+### substatus
+A status can mean multiple things, so substatus gives more details about the status. For example, an issue status can change to `unresolved` because it is now `escalating` or because a resolved issue is `regressed`. Possible substatuses are: `archived_until_escalating`, `archived_until_condition_met`, `archived_forever`, `escalating`, `ongoing`, `regressed` and `new`.
+
+### statusDetails
+`statusDetails` includes more detailed information about the action being done on the issue, for example is the issue is being resolved or archived. These details are:
+* `inRelease`: The release that the resolution is in for example `latest`
+* `inNextRelease`: `True` if the resolution will be in the next release
+* `inCommit`: Includes information about the resolution commit, for example `repository` and `commit`
+* `ignoreDuration`: Duration for an archived issue before it's turns to escalated in minutes
+* `ignoreCount`: Maximum number of occurences in `ignoreWindow` minutes for an archived issue before it gets escalated                                 
+* `ignoreUserCount`: Maximun number of users in `ignoreUserWindow` minutes who are affected by an archived issue before it gets escalated
```

Comment:
Where do `ignoreWindow` and `ignoreUserWindow` come from? Are they also part of statusDetails? Can we add them to this list (before ignoreCount and ignoreUserCount)? 

```suggestion
* `ignoreCount`: Maximum number of occurrences that can happen within the duration of `ignoreWindow` for an archived issue before it gets escalated                                 
* `ignoreUserCount`: Maximum number of users who can be affected by an archived issue within the duration of `ignoreUserWindow` before it gets escalated
```

Reply:
Yes, they're part of the same map. My thinking was because these don't make sense alone I should document them together. I'll add them